tasks/DroneFlightControlTask.cpp

```diff
@@ -49,7 +49,7 @@ bool DroneFlightControlTask::startHook()
     mLastMission = Mission();
 
     // Obtain Control Authority
-    mVehicle->obtainCtrlAuthority(mFunctionTimeout);
+    mAuthorityStatus = mVehicle->obtainCtrlAuthority(mFunctionTimeout);
```

Comment:
Shouldnt we just return false if we failed to get authority? Maybe that would tamper with telemetry